docs/quickstart/clustering.md

```diff
@@ -0,0 +1,129 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.1
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# Clustering
+
+In this guide, you'll learn how to use `sklearn-evaluation`, and `sklearn` to evaluate clustering models.
+
++++
+
+## Sample clustering model
+
+Let's generate some sample data with 5 clusters; note that in most real-world use cases, you won't have ground truth data labels (which cluster a given observation belongs to). However, in this case, the ground truth data is available, which will help us explain the concepts more clearly.
+
+```{code-cell} ipython3
+from sklearn.datasets import make_blobs
+
+X, y = make_blobs(n_samples=1000,
+                  centers=5,
+                  n_features=20,
+                  random_state=0,
+                  cluster_std=3,
+                  center_box=(-10, 10))
+```
+
+## Visualizing clusters
+
+Visualizing high-dimensional data is difficult. A common approach is to reduce its dimensionality using PCA; this losses some information but can help us visualize the clusters. Let's run PCA on our data and plot it:
+
+```{code-cell} ipython3
+import matplotlib.pyplot as plt
+from sklearn.decomposition import PCA
+
+pca = PCA(n_components=2)
+X_pca = pca.fit_transform(X)
+
+plt.scatter(X_pca[:, 0], X_pca[:, 1], c=y)
+plt.grid()
+_ = plt.title("Data in PCA space")
```

Comment:
The plot has no labels, it's unclear what's in X and Y

Reply:
will fix it